app/components/recognition/form.hbs

```diff
@@ -159,29 +166,91 @@
               }}
                 <Item>
                   <:label>
-                    <AuLabel @inline={{true}} for="legal-resource">Erkenningsbesluit</AuLabel>
+                    <AuLabel
+                      @inline={{true}}
+                      for="legal-resource"
+                    >Erkenningsbesluit</AuLabel>
                   </:label>
                   <:content>
-                    <AuInput
-                      placeholder="Voeg alleen een directe link naar het besluit toe"
-                      {{on "input" (fn this.clearFormError "legalResource")}}
-                      {{on "input" (event-value (fn (mut model.legalResource)))}}
-                      value={{model.legalResource}}
-                      @width="block"
-                      id="legal-resource"
-                      @error={{this.validationErrors.legalResource}}
-                    />
+                    {{#if this.currentRecognition.file}}
+                      <AuInput
+                        placeholder="Voeg alleen een pdf toe"
+                        value={{this.currentRecognition.file.name}}
+                        @width={{"block"}}
+                        readonly
+                        id="legal-resource"
+                        @error={{this.validationErrors.legalResource}}
+                      />
+                    {{else}}
+                      <AuInput
+                        placeholder="Voeg alleen een directe link naar het besluit toe"
+                        {{on
+                          "input"
+                          (event-value (fn (mut model.legalResource)))
+                        }}
+                        {{on "input" (fn this.clearFormError "legalResource")}}
+                        @width={{"block"}}
```

Comment:
Nitpick, I just noticed this was changed. Did you change this manually, or was this done by a formatter?

```diff
- @width={{"block"}}
+ @width="block"
```

If it was done manually, I wouldn't do that for strings in the future. The extra curlies aren't needed. If it's done by a formatter I would try to find out why it does it, because it shouldn't :smile: .